src/main/java/ch/admin/bag/covidcertificate/domain/DisplayNameModificationRepository.java

```diff
@@ -0,0 +1,18 @@
+package ch.admin.bag.covidcertificate.domain;
+
+import ch.admin.bag.covidcertificate.domain.enums.EntityType;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
+import org.springframework.stereotype.Repository;
+
+import java.util.Optional;
+
+@Repository
+public interface DisplayNameModificationRepository extends JpaRepository<DisplayNameModification, String> {
+
+    @Query(value = "select dnm from DisplayNameModification dnm " +
+            "where dnm.code = :code " +
+            "and dnm.entityType = :entityType ")
+    Optional<DisplayNameModification> findByCodeAndEntityType(@Param("code") String code, @Param("entityType") EntityType entityType);
```

Comment:
jpa does not need a query here - findByCodeAndEntityType gets translated natively to the right query (see https://www.bezkoder.com/jpa-repository-query/#JPA_find_by_multiple_Columns)